code-challenges.test.js

```diff
@@ -14,47 +14,148 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+describe ("fibCount", () => {
+  it("takes in a number > 2, and returns an array with a length equal to the number containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    
+    const fibLength2 = 10
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(fibCount(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibCount(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+// fibCount
+//   ✕ takes in a number > 2, and returns an array with a length equal to the number containing the numbers of the Fibonacci sequence (1 ms)
+
+// ● fibCount › takes in a number > 2, and returns an array with a length equal to the number containing the numbers of the Fibonacci sequence
+
+//   ReferenceError: fibCount is not defined
+
 
 // b) Create the function that makes the test pass.
+//  Pseudo Code
+//  Create a fuction called fibCount
+//  Input: a number
+//  Output: an array of fibonacci sequece
+//  Declare a variable called fibArr to store an array containing part of the Fibonacci sequence.
+//  In order for the function engage more outputs, I think a for loop will be needed in order to iterate or in other words
+//  Pass a condition that if num is less then or equal to 0 then the return is [0]
+//  Pass a condition that if num is equal to 1 then return [1]
+//  
+//    What if is Fib. Sequence?:  0 + 1 = 1
+//                                1 + 1 = 2 
+//                                1 + 2 = 3
+//                                2 + 3 = 5
+//                                3 + 5 = 8
+//                                5 + 8 = 13
+// 
+
+const fibCount = (num) => {
+  if (num <= 0) {
+    return []
+  }
+  if (num === 1) {
+    return [1]
+  }
+  var fibArr =[1,1]
+  for (var i = 2; i < num; i++) {
+    fibArr[i] = fibArr[i - 1] + fibArr[i - 2]
+  }
+  return fibArr.slice(0, num)
```

Comment:
This is a great start! Your logic looks good. What is the intention with slice method here?